scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -599,9 +608,7 @@ def log_event(self, event):
 
         if "TimeCreated" in vals:
             time_format = "%Y-%m-%d %H:%M:%SZ"
-            vals["TimeCreated"] = time.strftime(
-                time_format, time.gmtime(int(vals["TimeCreated"]))
-            )
+            vals["TimeCreated"] = vals["TimeCreated"].strftime(time_format)
```

Comment:
To confirm, ``vals["TimeCreated"]`` is a datetime object, right?

Reply:
Is this a change in recent pywin32 version, or? (since it looks like it used to be a timestamp / number, right?)

Reply:
It should be a change in the `pywin32`. Yes, it used to be an int but it seems that they introduced their own datetime object and now it is used there.